packages/fxa-auth-server/lib/google-maps-services.ts

```diff
@@ -64,8 +64,23 @@ export class GoogleMapsService {
     if (status === Status.ZERO_RESULTS)
       throw new Error(`Could not find any results for address. (${address})`);
 
-    if (results.length > 1)
-      throw new Error(`Could not find unique results. (${address})`);
+    if (results.length > 1) {
+      const stateShortNames = results.map(
```

Comment:
Before `.map` could we also `.filter` on results where `PlaceType2.postal_code` actually equals the input postal code?

For example with postal code 10000 for the United States, none of the results returned actually have a `PlaceType2.postal_code` that equals 10000.

This was actually a bit of an oversight during initial implementation.

Reply:
Sure I changed this, though for the specific example of 10000, the result is the same (i.e. we throw an error anyway, since one of the `results` did not have a state in its `addressComponents` list).

Reply:
So weird to get a result with a non-matching postal code...